### ISSUE TITLE
New import message format

### ISSUE THREAD

**Issue by IanMayo** [debrief/debrief#4353]
An analyst (SE) wishes to load track data from another source.  

The source can output multiple tracks of data at a time in the OTH-Gold format.

View the data, produce a demo version of the data, to develop against, then trial it.

**Comment by IanMayo** [comment]
Sample, of multiple track contact report (from U/C spec dated 2007):
````
MSGID/CTG81.0/GOLD/1215/APR/RIMPAC 00 CTC/T7123/UNEQUATED-UNKNOWN//SSGN/SUB//XX///HIT/14//3/10/10 /3442/F2K000162412/CAOB/03 POS/121410Z9/APR/2540N1/16204W3/VISUAL//10NM//340T/12K RADB/121410Z9/APR/0923B/FRONT PACE/12.34GHZ/000123.45/8100.446 /123.123/5.0SPC/CIRC
EOB//D18445994 SIGNA/XXX/123.45HZ/6,10,4/88/10.5/121410Z9/APR/VDSACT/STEADY SIGAM/UNIDENTIFIED SIGNAL IN MAIN MACHINERY SPACE MODEG/SUBMERGED/OTR/850FT/E/IN-LAYER DECIBELS CTC/T7164/UNEQUATED-UNKNOWN/TRDMARK/SSN/SUB//////00/3447/3 LOB/121420Z0/APR/2510N0/16210W0/120T/TACTAS/1.2DEG RADB/121420Z0/APR/0924Z/TRAP DEER/13.20GHZ/000200.55/4987.531/222.001 RMKS/EVALUATED AS TYPE II NUC
ENDAT
````

The 3rd party supplier clarified:
```
Our system only produces the sets listed below.
MSGID
CTC
POS
RMKS
ENDAT
```

So a streamlined version of the original example would be:
````
MSGID/CTG85.0/GOLD/1215/APR/DANPAC 00 
CTC/T7123/UNEQUATED-UNKNOWN//SSGN/SUB//XX///HIT/14//3/10/10/3442/F2K000162412/CAOB/03 
POS/121410Z9/APR/2540N1/16204W3/VISUAL//10NM//340T/12K 
POS/121415Z4/APR/2541N2/16206W5/VISUAL//11NM//20T/14K 
RMKS/EVALUATED AS TYPE WHISKY-NINE
CTC/T7164/UNEQUATED-UNKNOWN/TRDMARK/SSN/SUB//////00/3447/3 
POS/121411Z0/APR/2541N2/16006W3/VISUAL//11NM//120T/4K 
POS/121416Z5/APR/2543N3/16106W4/VISUAL//9NM//90T/6K 
RMKS/EVALUATED AS TYPE TWO-DELTA
ENDAT
````

**Comment by IanMayo** [comment]
Logic:
* register ".txt" file loader
* verify we can import it, by looping through until we find a line beginning with MSGID _AND_ a line beginning `POS`
* once import starts:
    * Take year from first line (if present)
    * loop forward to `CTC` line
        * extract platform name
        * loop through pos entries
            * extract new FIX (location,course,speed)
            * extract new ellipse
        * until another `CTC` or `ENDAT`
    * ask user whether ellipses should be created
    * create tracks
    * create ellipses (if user decided to)

**Comment by IanMayo** [comment]
A sample data file has been placed in `sample_data/other_formats/OTH_Gold.txt`

**Comment by IanMayo** [comment]
Closed in #4466
